asteroid/modules/prologue.ast

```diff
@@ -973,3 +973,21 @@ with none do
                 @reverse()
                 @join("").
 end
+
+------------------------------------------------------------------
+--Repl autocompletion toggle
+------------------------------------------------------------------
+escape
+"
+import os
+import sys
+ 
+file_path = os.path.dirname(os.path.abspath( __file__ ))
+os.chdir(file_path)
+(parent_dir,_) = os.path.split(file_path)
+sys.path.append(parent_dir)
+
+from lex import set_repl_autocomplete
+
+set_repl_autocomplete(False)
```

Comment:
I am starting to think the reason this is not working is that your sandbox is not setup correctly.